web/simonamarcu2/html/primul/simy.css

```diff
@@ -1,43 +1,68 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@200&family=Poppins:wght@100;500&display=swap");
+
 body {
   background-color: lavender;
-  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+  font-family: "Montserrat", sans-serif;
   font-size: larger;
 }
 
-h1 {
-  background-color: aquamarine;
+.member {
+  display: inline-flexbox;
+  border-radius: 5px;
+  color: whitesmoke;
+  font-size: 20px;
+  padding: 5px 10px;
+  background-color: purple;
   text-align: center;
-  border: 4px solid violet font family cursive;
+  font-family: "Poppins", sans-serif;
+  letter-spacing: 7px;
+  font-weight: 700;
+}
+
+a {
+  text-decoration: none;
+}
+
+h1 {
+  border-radius: 3px;
+  /* border: 4x solid rgb(20, 114, 177); */
+  display: block;
+  width: 250px;
+  padding: 10px;
+  margin: 0 auto;
+  background-color: rgb(171, 214, 231);
 }
 
 h2 {
-  background-color: hsl(125, 54%, 73%);
   text-align: center;
-  font-family: cursive;
+  background-blend-mode: soft-light;
+  background-color: rgb(183, 243, 218);
 }
 
-h3 {
+/* h3 {
   text-align: center;
-  font-family: cursive;
-}
+} */
 
 p {
-  text-align: center;
-  font-family: sans-serif;
+  text-align: left;
+  text-decoration: lavender;
```

Comment:
Maybe you wanted to say: `text-decoration-color: lavender;` ?

Reply:
Thank you for observation!